### ISSUE TITLE
Error in git --help description

### ISSUE THREAD

**Issue by massimosgrelli** [secrez/secrez#138]
I think there's an error in the description of git command:
```
Examples:
  git -p      (adds, commits and pushes to the remote repo; a message is not allowed because it is better not to give
                any help to a possible hacker about whatever you are committing)
  git --push  (pulls from origin and merges)
  git -i      (get info about the repo, like the remote url)
```
its should be `git --pull `or `git -P` and not `git --push`.

**Comment by sullof** [comment]
You are right. I will fix it asap.

**Comment by sullof** [comment]
Fixed in 0.10.5
